### PR TITLE
invoices: fix log messages

### PR DESCRIPTION
**Pull request by bhandras**
This PR fixes incorrect log messages.

### CODE REVIEW

File: invoices/invoice_expiry_watcher.go [hunk 1]
```diff
@@ -139,8 +139,9 @@ func (ew *InvoiceExpiryWatcher) AddInvoices(
 	}
 
 	if len(invoicesWithExpiry) > 0 {
-		log.Debugf("Added %v invoices to the expiry watcher: %v",
-			len(invoicesWithExpiry))
+		log.Debugf("Added %v invoices to the expiry watcher",
```
Comment (guggero) [inline invoices/invoice_expiry_watcher.go]: nit: Could be `%d` as we know it's numeric. And I think the logging and error message construction calls are the only exceptions where we don't put the closing parenthesis on a new line.

Comment (bhandras) [reply invoices/invoice_expiry_watcher.go]: fixed

**Comment by halseth** [comment]
@bhandras Is this still a draft PR?

**Comment by bhandras** [comment]
> @bhandras Is this still a draft PR?

No, sorry forgot to push the button.

**Pull request merged**